packages/radio-group/src/vaadin-radio-button.js

```diff
@@ -151,6 +151,21 @@ class RadioButton extends SlotLabelMixin(
     return this.$.noop;
   }
 
+  /**
+   * Override __copyNodesToSlotTarget from SlotTargetMixin to show a warning.
+   * @override
+   * @protected
+   * @param {Array<Node>} nodes
+   **/
+  __copyNodesToSlotTarget(nodes) {
+    super.__copyNodesToSlotTarget(nodes);
+
+    console.warn(
+      `WARNING: Since Vaadin 22, placing the label as a direct child of a <vaadin-radio-button> is deprecated.
+  Please use <label slot="label"> wrapper or the "label"-property instead.`
```

Comment:
```suggestion
      `WARNING: Since Vaadin 22, placing the label as a direct child of a <vaadin-radio-button> is deprecated.
  Please use <label slot="label"> wrapper or the label property instead.`
```

```suggestion
      `WARNING: Since Vaadin 22, placing the label as a direct child of a <vaadin-radio-button> is deprecated.
  Please use <label slot="label"> wrapper or the "label" property instead.`
```